components/experimental/src/units/ratio.rs

```diff
@@ -432,6 +437,21 @@ impl FromStr for IcuRatio {
 mod tests {
     use super::*;
 
+    #[test]
+    fn test_ratio_always_keep_sign_in_numerator() {
```

Comment:
remove?

Reply:
sure, in the following PR

Reply:
https://github.com/unicode-org/icu4x/pull/4835